byron-proxy/src/exec/IPC.hs

```diff
@@ -1,37 +1,33 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE OverloadedStrings #-}
 
 module IPC where
```

Comment:
Changes in here just get rid of the prototype versioning protocol I had implemented. We'll replace it with the one in #425 later